arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790546.aspx"
+          },
+          "type": "string",
+          "description": "Gets or sets the resource group name."
+        },
+        "resourceProviderName": {
+          "$ref": "#/definitions/LocalizableString",
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790572.aspx"
+          },
+          "description": "Gets or sets the resource provider name."
+        },
+        "resourceId": {
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790569.aspx"
+          },
+          "type": "string",
+          "description": "Gets or sets the resource uri"
+        },
+        "resourceType": {
+          "$ref": "#/definitions/LocalizableString",
+          "externalDocs": {
+            "url": "http://msdn.microsoft.com/en-us/library/azure/dn790569.aspx"
+          },
+          "description": "Gets or sets the resource type"
+        },
+        "operationId": {
+          "type": "string",
+          "description": "Gets or sets the operation idThis value should not be confused with EventName."
+        },
+        "operationName": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the operation name."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the property bag"
+        },
+        "status": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the event status.Some typical values are: Started, Succeeded, Failed"
+        },
+        "subStatus": {
+          "$ref": "#/definitions/LocalizableString",
+          "description": "Gets or sets the event sub status.Most of the time, when included, this captures the HTTP status code."
+        },
+        "eventTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the occurrence time of event"
+        },
+        "submissionTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the event submission time.This value should not be confused eventTimestamp. As there might be a delay between the occurence time of the event, and the time that the event is submitted to the Azure logging infrastructure."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "Gets or sets the Azure subscription Id"
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "Gets or sets the Azure tenant Id"
+        }
+      },
+      "description": "The Azure event log entries are of type EventData"
+    },
+    "EventDataCollection": {
+      "properties": {
+        "value": {
```

Comment:
Operations that return an EventDataCollection have x-ms-pageable extension defined with a nextLink property. However the model does not have nextLink defined.